client/src/actions/index.js

```diff
@@ -0,0 +1,35 @@
+import axios from "axios";
```

Comment:
Why are we using this instead of just fetch? Just curious

Reply:
I don't know. It was already in package.json -- that was my only reason. I'll change to fetch.